lib/error/error_manager_impl.cpp

```diff
@@ -81,7 +83,8 @@ std::list<ErrorPtr> ErrorManagerImpl::clear_error(const ErrorType& type, const E
     std::list<ErrorFilter> filters = {ErrorFilter(TypeFilter(type)), ErrorFilter(SubTypeFilter(sub_type))};
     std::list<ErrorPtr> res = database->remove_errors(filters);
     if (res.size() > 1) {
-        throw EverestBaseLogicError("There are more than one matching error, this is not valid");
+        EVLOG_error << "There are more than one matching error, this is not valid";
+        return {};
```

Comment:
To ignore this case it should be checked with `database->get_errors(filters)` first, to then don't remove .

Anyways this means that the database is in a corrupted state which is not recoverable, so it should crash/throw

Reply:
It should not, see above. The problem is that a crash does not solve the problem, after restart, it is still there and will crash again, ending up in a restart loop. If it can be guaranteed that it never happens, it does not need to be checked here. Apparently it can happen, and then it needs to be dealt with somehow, e.g. removing one of them in the database.